wiki/Game_mode/id.md

```diff
@@ -0,0 +1,46 @@
+# Mode permainan
+
+osu! memiliki empat **mode permainan** yang dapat kamu mainkan: [osu!](#-osu!), [osu!taiko](#-osu!taiko), [osu!catch](#-osu!catch), dan [osu!mania](#-osu!mania). Keempat mode permainan tersebut memiliki mekanisme permainannya masing-masing yang terinspirasi dari berbagai permainan ritme lainnya yang ada di luar sana.
+
+Setiap mode permainan memiliki simbol/ikonnya masing-masing sebagai berikut: ![][o!s] ![][o!t] ![][o!c] ![][o!m] (yang secara berurutan melambangkan mode osu!, osu!taiko, osu!catch, dan osu!mania).
+
+Secara *default*, kamu akan disambut dengan mode permainan yang terakhir kali kamu mainkan setiap kali kamu membuka klien osu! milikmu. Untuk dapat berpindah dari satu mode permainan ke mode permainan lainnya, kamu dapat menggunakan tombol `Mode` yang berada pada bagian bawah layar Song Selection atau menekan kombinasi tombol `Ctrl`+`1`, `Ctrl`+`2`, `Ctrl`+`3`, atau `Ctrl`+`4` untuk berpindah ke mode osu!, osu!taiko, osu!catch, atau osu!mania.
```

Comment:
```suggestion
Secara *default*, kamu akan disambut oleh mode permainan yang terakhir kali kamu mainkan setiap kali membuka klien osu!. Untuk dapat berpindah dari satu mode permainan ke mode permainan lainnya, kamu dapat menggunakan tombol `Mode` yang berada pada bagian bawah layar Song Selection atau menekan kombinasi tombol `Ctrl`+`1`, `Ctrl`+`2`, `Ctrl`+`3`, atau `Ctrl`+`4` untuk berpindah ke mode osu!, osu!taiko, osu!catch, atau osu!mania.
```

Reply:
`oleh` =x `dengan` because not in pairs.
fix using too much `kamu` glued misunderstanding.

Reply:
did sth else here